src/components/book/book.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { removeBook, changeStatus } from '../../redux/actions/library';
+import './book.styles.scss';
+
+const Book = ({id, title, category, read, removeBook, changeStatus }) => {
+
+	const handleDelete = (id) => {
```

Comment:
Once more, we shouldn't be doing data fetching logic here. See my other comments in BookForm and sign-in